dynacred/tsconfig.json

```diff
@@ -1,17 +1,36 @@
 {
   "compilerOptions": {
+    "module": "commonjs",
```

Comment:
ES6 as real support for module, no need to explicit `"module"`

Reply:
OK, I'll have another look at these `tsconfig.json` options - it was mainly a manual effort to get `dynacred`, `@dedis/cothority`, `crypto`, `nativescript-nodeify` all working together nicely.

Reply:
that's the only one that I need - else `npm run build` fails.

Reply:
Yep, there is a relation between `target` and `module` (I prefer the former as going forward as ES6 as support for module).
https://www.typescriptlang.org/docs/handbook/compiler-options.html

Reply:
As far as I understand, this is how the modules are resolved. So if I remove that, it doesn't find any modules anymore.